pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "qiskit[all]~=0.39.2",
     "pytket~=1.9.0",
     "mqt.bench@ git+https://github.com/cda-tum/MQTbench@dev",
+    "sb3-contrib>=1.7.0a0",
```

Comment:
I might be stupid, bur aren't we missing the `gym` dependency here?

Reply:
`gym` is part of `stable-baselines3` or `sb3-contrib`. To avoid version confusions, I removed the explicit `gym` version in our project.

Reply:
Ah. I like that. One dependency less 👍🏻